setup.py

```diff
@@ -23,6 +23,12 @@
         "pytorch-sphinx-theme==0.0.19",
         "sphinxcontrib-versioning",
         "Jinja2<3.1",
+        "neuromllite>=0.5.1",
+        "pyNeuroML>=0.5.20",
```

Comment:
I think these should be broken apart into individual extra requirements for each backend, Something like:
```
extras = {
    "psyneulink": ["psyneulink>=0.12.0.0"],
    "neuroml": ["pyNeuroML>=0.5.20", "neuromllite>=0.5.1"],
 ...

```       

We can then have `optional` be all core plus all backed requirements.